ui/imports/shared/views/chat/InvitationBubbleView.qml

```diff
@@ -177,7 +177,7 @@ Control {
             radius: d.radius - 1 // We do -1, otherwise there's a gap between border and button
 
             onClicked: {
-                if (d.invitedCommunity.joined || d.invitedCommunity.spectated) {
+                if ((d.invitedCommunity.joined || d.invitedCommunity.spectated) && !d.invitedCommunity.amIBanned) {
```

Comment:
Why we don't go to the community when banned? It will make "go to community" not working.

https://user-images.githubusercontent.com/33099791/208405277-a2489192-adf5-4256-b2da-0c68a9d5b391.mp4

Reply:
I agree with Patryk, you can leave this one active.